### PR TITLE
Bugfix: move common functionality to utils.py

### PR DESCRIPTION
**Pull request by ashwini-mhatre**
##### SUMMARY

fixes: https://github.com/ansible-collections/cisco.iosxr/issues/113

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

iosxr_bgp_global, iosxr_address_family, iosxr_neighbor_address_family

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/fragments/115_iosxr_move_common_functionality_to_util.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+bugfixes:
```
Comment (NilashishC) [inline changelogs/fragments/115_iosxr_move_common_functionality_to_util.yaml:2]: This is probably a trivial change? If I understand correctly, this optimizes the code internally and is not something that affects the end user experience.

Comment (NilashishC) [reply changelogs/fragments/115_iosxr_move_common_functionality_to_util.yaml:2]: The code changes look fine to me.

Comment (ashwini-mhatre) [reply changelogs/fragments/115_iosxr_move_common_functionality_to_util.yaml:2]: okay will add changelog as trivial

Comment (ashwini-mhatre) [reply changelogs/fragments/115_iosxr_move_common_functionality_to_util.yaml]: ack

**Review by NilashishC — APPROVED** [review]

**Pull request merged**